Automattic/wp-calypso
# Manage Purchase: Add tax (Placeholder Edition)

#### Changes proposed in this Pull Request

This PR adds tax placeholders to the "Manage Purchases" page under `/me/purchases/$Site/$PurchaseID`

#### Testing instructions

 - Navigate to calypso.localhost:3000/me/purchases/billing and click the 'Manage Purchase' link under some of you Upcoming Charges.
- Verify no tax is shown.
- Add `?flags=show-tax,tax-placeholders` to the url (or enable the two flags in your local config)
- Verify tax is shown

![purchases_ _manage_purchase_ _wordpress_com](https://user-images.githubusercontent.com/5952255/49983780-7231e680-ffb0-11e8-8274-415d27734d92.jpg)

You can also confirm that the tax amount is getting injected in the console with:
`_.filter( getState().purchases.data, 'tax_text' )`, or with `localStorage.setItem( 'debug', localStorage.debug + '.calypso:tax-placeholders' )`
